### DISCUSSION TITLE
z2m does not start after reboot

### DISCUSSION THREAD

**Discussion by NormKOALA** [Koenkk/zigbee2mqtt#7000] [General]
After migration to slaesh stick CC2652RB, first time z2m works fine (zigbee devices are added quickly and works correct), but after rebooting of system or z2m add-on z2m can not start.
(To be fair, it starts, but after about 10 minutes and without early added devices.)

And the same result when I tried reinstalling mqtt and z2m.
Also I tried reflashing stick and using power hub, no changes.
I have changed panID, changed serial port and etc and etc....

Oh! It is restarted without any problems when no added zigbee devices....

**Comment by Koenkk** [comment]
Seems the same as https://github.com/Koenkk/zigbee2mqtt/issues/6302#issuecomment-779394499

**Reply by NormKOALA** [reply]
Thank you for your answer, but I saw this thread before writing here, and it does not help.
Initially I had 2 sticks that were flashed by Slaesh, and one of them I reflashed myself. Result is same.

**Reply by Koenkk** [reply]
Did you also reflash on a totally different system?
